Description: Work around predefined "pixel" on some architectures
 "pixel" is predefined on some architectures, confusing gtk2hsC2hs.
 Explicitly undefine it to work around this.
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: no
Last-Update: 2014-12-03

Index: haskell-gtk3/gtk3.cabal
===================================================================
--- haskell-gtk3.orig/gtk3.cabal	2015-08-15 14:35:59.670399380 +0200
+++ haskell-gtk3/gtk3.cabal	2015-08-15 14:35:59.666399298 +0200
@@ -385,6 +385,8 @@
 
         pkgconfig-depends: gthread-2.0, gtk+-3.0
 
+        cpp-options: -Upixel
+
 Executable gtk2hs-demo-actionMenu
     default-language: Haskell98
     hs-source-dirs: demo/actionMenu
